proposed/2021/ShowVulnerabilitiesInThePMUI.md

```diff
@@ -0,0 +1,90 @@
+# Show NuGet Package Vulnerabilities in the Visual Studio Package Manager UI
+
+* Status: **MVP Implementation**
+* Author(s): [Chris Gill](https://github.com/chgill-MSFT)
+* Issue: 
+* Type: Feature
+
+## Description: What is it?
+
+When a vulnerability in a NuGet package is discovered, surface an indicator of the vulnerability in the Package Manager UI to alert developers and help them take the appropriate action.
+
+## Problem: What problem is this solving?
+
+Vulnerabilities in packages can be leveraged by malicious actors to do harm to developers and their users. At the time of writing this, there are 113 NuGet advisories in the [GitHub Advisory Database](https://github.com/advisories?query=ecosystem%3Anuget). While package vulnerabilities can be found on NuGet.org and with the dotnet list command, most developer interactions with NuGet occur in the Visual Studio package manager UI where no vulnerability information is currently available.
+
+## Why: How do we know this is a real problem worth solving?
+
+NuGet hosts over 200,000 packages that developers use today. Most packages depend on another package, to which the attack vector / blast radius of any security vulnerability can be catastrophic in any package management ecosystem.
+
+## Success: How do we know if we've solved this problem?
+
+We will use the following to measure the success of this feature:
+
+* We see a significant decrease in direct installations for vulnerable packages versions after surfacing vulnerability indicators in the PMUI.
```

Comment:
Customers should upgrade their direct dependencies if their transitive dependencies have vulnerabilities. Is this something we want to track? Should we track the number of downloads on packages with vulnerabilities?

Reply:
Ah didn't see the section below for out of scope:

> Users can see which transitively installed packages have vulnerabilities.